src/Chapter15/Listing15.22.CreatingAChildCollectionWithSystem.Linq.Enumerable.GroupJoin.cs

```diff
@@ -1,18 +1,23 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter15.Listing15_22
 {
     using AddisonWesley.Michaelis.EssentialCSharp.Chapter15.Listing15;
+    #region INCLUDE
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    #region EXCLUDE
 
     public class Program
     {
         public static void Main()
         {
+            #endregion EXCLUDE
             Department[] departments = CorporateData.Departments;
             Employee[] employees = CorporateData.Employees;
 
-            IEnumerable<(long Id, string Name, IEnumerable<Employee> Employees)> items =
+            IEnumerable<
+                (long Id, string Name, IEnumerable<Employee> Employees)> 
+                    items =
```

Comment:
This seems odd to be spread over 3 lines to me

Reply:
Too long otherwise